examples/graphene_apps/python_worker/README.md

```diff
@@ -164,6 +164,22 @@ https://creativecommons.org/licenses/by/4.0/
           *"class": "<workload implementation class name>"*
       *}*
 
+## Testing OpenVino Object detection with Python worker
+
+- First build OpenVino docker image and GSC docker image using the steps mentioned in cppopenvino [ReadMe](https://github.com/hyperledger/avalon/blob/master/examples/graphene_apps/cppopenvino/README.md)
+
+- Below steps will run sample openvino inference test cases in [test_ov_work_orders.json](http://github.com/hyperledger/avalon/tree/master/examples/graphene_apps/python_worker/tests/test_ov_work_orders.json)
+
+  ### Testing OpenVino (without Avalon)
+
+  - For non SGX mode, execute below command
+
+    `docker-compose -f docker-compose.yaml -f compose/ov.yaml -f compose/ov-subnet.yaml up`
```

Comment:
Would be good to use docker/compose rather than compose only. Would give more context.